WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutPickerViewController.swift

```diff
@@ -56,6 +56,7 @@ class GutenbergLayoutPickerViewController: UIViewController {
         closeButton.setImage(UIImage.gridicon(.crossSmall), for: .normal)
         styleButtons()
         layoutHeader()
+        navigationController?.navigationItem.rightBarButtonItem?.accessibilityLabel = NSLocalizedString("Close", comment: "Close the modal window")
```

Comment:
This instance of `rightBarButtonItem` is `nil`, so this line is not actually setting the localized accessibility label.
What we want is to find the button on the `GutenbergLayoutPickerViewController`'s instance of `navigationItem`.

Each Controller manages their own navigation items.

This currently works because the label is also set via Storyboard, so it's a bit confusing, but won't be localized.

Reply:
> This currently works because the label is also set via Storyboard, so it's a bit confusing, but won't be localized.

Actually I don't think the storyboard Accessibility was being determined. Without this line VoiceOver reads "Button Possibly Close" however with this line it reads as "Close Button". 

However, with the changes @iamthomasbishop requested, I'll need to move this out of the navigation controller to make sure the Visual Effect line up. So it'll go away after I push those changes.

Reply:
> Without this line VoiceOver reads "Button Possibly Close" however with this line it reads as "Close Button".

Hah, Interesting! I did the exact same test (and just did it again now to be sure) and in both cases (with or without the line) VoiceOver reads it as "Close".

I'm using an iPhone 7 device.

Also `navigationController?.navigationItem.rightBarButtonItem` is nil to me. So ¯\\_(ツ)_/¯ 

> So it'll go away after I push those changes.

Ok then! 👍

Reply:
Very interesting! I'm on an iPhone 11 running 13.5 so I wonder if there was a change there although the release notes don't have anything. Good to know though!

Reply:
Just closing the loop on this one because I thought it was interesting. I retested and I'm getting the same results you are. So I probably had a build issue that was using a cached storyboard.